var/spack/repos/builtin/packages/gdbm/package.py

```diff
@@ -45,4 +45,25 @@ class Gdbm(AutotoolsPackage):
     depends_on("readline")
 
     def configure_args(self):
-        return ['--enable-libgdbm-compat']
+        spec = self.spec
+
+        # Environment variables
+        CFLAGS   = []
+        CPPFLAGS = []
+        LDFLAGS  = []
+        LIBS     = []
+
+        CPPFLAGS.append("-I%s/include" % spec['readline'].prefix)
```

Comment:
This is already done by the compiler wrappers. Does it work if you remove it?

Reply:
Well, the `develop` branch does not have this, and it does not work. As I commented in https://github.com/LLNL/spack/issues/5892#issuecomment-338813623, the configure script does *not* use the compiler wrappers.

Reply:
So an alterantive fix is to use compiler wrappers. How do you do that?

Reply:
https://spack.readthedocs.io/en/latest/packaging_guide.html#compiler-wrappers

The whole point of Spack builds is to inject the compiler wrappers in the build so that you can swap compilers and add flags like `-I`, `-L`, as well as flags specified on the command line, automatically.  By default we set `CC`, `CXX`, `FC` etc. so that most build systems will pick them up.  The wrappers also add `RPATH`s automatically for most builds and generally don't require any configuration to get working.

If the `configure` script doesn't use them, then you should figure out how to get them in there.  It *looks* like in this case you could just pass `'CC=%s' % spack_cc` on the command line.